ArraysHomework/ArraysHomework.cs

```diff
@@ -90,34 +100,39 @@ public void Arrays()
         public void Lists()
         {
             // Create a List of ints named my list
+            List<int> myList = new List<int>();
 
             Assert.IsTrue(myList != null); // if you create it right, this will pass
 
             // Don't worry about how this line works. It's not something you need at this point.
             // All it's doing is making sure that you actually created a List of ints (rather than, for example, an array of ints).
-            Assert.IsTrue(myList.GetType() == typeof(List<int>));   
+            Assert.IsTrue(myList.GetType() == typeof(List<int>));
 
             // take the list made above and fill it so these test passes  
+            myList.Add(21);
+            myList.Add(35);
+            myList.Add(78);
 
             Assert.IsTrue(myList[0] == 21);
             Assert.IsTrue(myList[1] == 35);
             Assert.IsTrue(myList[2] == 78);
 
             // what wrong with this list declaration? You can use any type for its content. Fix it and make it compile
-            List<> stringList = List;
+            List<string> stringList = new List<string>();
 
             // What about this one? You can use any type for its content. Fix it and make it compile
-            List<> list1 = List<>();
+            List<int> list1 = new List<int>();
 
             // What about this one? You can use any type for its content. Fix it and make it compile
-            List<> list2 = new;
+            List<float> list2 = new List<float>();
 
             // What about this one? You can use any type for its content. Fix it and make it compile
-            List<char> list3 = new List<int>;
+            List<char> list3 = new List<char>();
 
             // Take this list and copy its contents into a new list 
             List<char> copyMe = new List<char>() { 'a', 'b', 'c' };
-
+            List<char> copiedList = new List<char>();
+            copiedList = copyMe;
```

Comment:
This also creates a reference to the same List. How can you copy the values from one list to the other?